### PR TITLE
Add info for sample gallery

### PR DESCRIPTION
**Pull request by davidchesnut**
|        Q        |                    A                    |
| --------------- | --------------------------------------- |
| Bug fix?        | no                                |
| New feature?    | no                                |
| New sample?     | no                                |
| Related issues? | none |

## What's in this Pull Request?

Add sample.json file to list sample in the sample gallery.

### CODE REVIEW

**Comment by aafvstam** [comment]
Should I also add these to the Word and Excel Blazor Sample to keep the three in sync?

File: README.md [hunk 1]
```diff
@@ -17,6 +17,8 @@ The following samples show how to build the simplest Office Add-in with only a m
 | -------------- | ------------ |
 | [Create a Blazor WebAssembly Excel add-in](https://github.com/OfficeDev/Office-Add-in-samples/tree/main/Samples/blazor-add-in/excel-blazor-add-in) | Uses .NET Blazor technologies to build an Excel add-in. |
 | [Create a Blazor WebAssembly Word add-in](https://github.com/OfficeDev/Office-Add-in-samples/tree/main/Samples/blazor-add-in/word-blazor-add-in) | Uses .NET Blazor technologies to build a Word add-in. |
+| [Create a Blazor WebAssembly Outlook add-in](https://github.com/OfficeDev/Office-Add-in-samples/tree/main/Samples/blazor-add-in/outlook-blazor-add-in) | Uses .NET Blazor technologies to build a Outlook add-in. |
```
Comment (samantharamon) [inline README.md:20]:
```suggestion
| [Create a Blazor WebAssembly Outlook add-in](https://github.com/OfficeDev/Office-Add-in-samples/tree/main/Samples/blazor-add-in/outlook-blazor-add-in) | Uses .NET Blazor technologies to build an Outlook add-in. |
```

File: Samples/blazor-add-in/excel-blazor-add-in/assets/sample.json [hunk 2]
```diff
@@ -0,0 +1,37 @@
+[
+    {
+      "name": "excel-blazor-add-in",
+      "reponame": "office-add-in-samples",
+      "source": "officedev",
+      "title": "Create a Blazor Webassembly Excel add-in",
```
Comment (samantharamon) [inline Samples/blazor-add-in/excel-blazor-add-in/assets/sample.json:6]:
```suggestion
      "title": "Create a Blazor WebAssembly Excel add-in",
```

File: Samples/blazor-add-in/excel-blazor-add-in/assets/sample.json [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+      "shortDescription": "This sample shows how to build an Excel add-in using .NET Blazor technologies.",
+      "url": "https://github.com/OfficeDev/Office-Add-in-samples/tree/main/Samples/blazor-add-in/outlook-blazor-add-in",
```
Comment (samantharamon) [inline Samples/blazor-add-in/excel-blazor-add-in/assets/sample.json:8]:
```suggestion
      "url": "https://github.com/OfficeDev/Office-Add-in-samples/tree/main/Samples/blazor-add-in/excel-blazor-add-in",
```

File: Samples/blazor-add-in/excel-blazor-add-in/assets/sample.json [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+      "longDescription": [
+        "This sample shows how to build an Excel add-in using .NET Blazor technologies. Blazor Webassembly allows you to build Office Add-ins using .NET, C#, and JavaScript to interact with the Office JS API. The add-in uses JavaScript to work with the document and Office JS APIs, but you build the user interface and all other non-Office interactions in C# and .NET Core Blazor technologies."
```
Comment (samantharamon) [inline Samples/blazor-add-in/excel-blazor-add-in/assets/sample.json:10]:
```suggestion
        "This sample shows how to build an Excel add-in using .NET Blazor technologies. Blazor WebAssembly allows you to build Office Add-ins using .NET, C#, and JavaScript to interact with the Office JS API. The add-in uses JavaScript to work with the document and Office JS APIs, but you build the user interface and all other non-Office interactions in C# and .NET Core Blazor technologies."
```

File: Samples/blazor-add-in/outlook-blazor-add-in/assets/sample.json [hunk 5]
```diff
@@ -0,0 +1,42 @@
+[
+    {
+      "name": "outlook-blazor-add-in",
+      "reponame": "office-add-in-samples",
+      "source": "officedev",
+      "title": "Create a Blazor Webassembly Outlook add-in",
```
Comment (samantharamon) [inline Samples/blazor-add-in/outlook-blazor-add-in/assets/sample.json:6]:
```suggestion
      "title": "Create a Blazor WebAssembly Outlook add-in",
```

File: Samples/blazor-add-in/outlook-blazor-add-in/assets/sample.json [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+      "shortDescription": "This sample shows how to build an Outlook add-in using .NET Blazor technologies.",
+      "url": "https://github.com/OfficeDev/Office-Add-in-samples/tree/main/Samples/blazor-add-in/outlook-blazor-add-in",
+      "longDescription": [
+        "This sample shows how to build an Outlook add-in using .NET Blazor technologies. Blazor Webassembly allows you to build Office Add-ins using .NET, C#, and JavaScript to interact with the Office JavaScript API. The add-in uses JavaScript to work with the document and Office JS APIs, but you build the user interface and all other non-Office interactions in C# and .NET Core Blazor technologies."
```
Comment (samantharamon) [inline Samples/blazor-add-in/outlook-blazor-add-in/assets/sample.json:10]:
```suggestion
        "This sample shows how to build an Outlook add-in using .NET Blazor technologies. Blazor WebAssembly allows you to build Office Add-ins using .NET, C#, and JavaScript to interact with the Office JavaScript API. The add-in uses JavaScript to work with the document and Office JS APIs, but you build the user interface and all other non-Office interactions in C# and .NET Core Blazor technologies."
```

File: Samples/blazor-add-in/word-blazor-add-in/assets/sample.json [hunk 7]
```diff
@@ -0,0 +1,37 @@
+[
+    {
+      "name": "word-blazor-add-in",
+      "reponame": "office-add-in-samples",
+      "source": "officedev",
+      "title": "Create a Blazor Webassembly Word add-in",
```
Comment (samantharamon) [inline Samples/blazor-add-in/word-blazor-add-in/assets/sample.json:6]:
```suggestion
      "title": "Create a Blazor WebAssembly Word add-in",
```

File: Samples/blazor-add-in/word-blazor-add-in/assets/sample.json [hunk 8]
```diff
@@ -0,0 +1,37 @@
...
+      "shortDescription": "This sample shows how to build a Word add-in using .NET Blazor technologies.",
+      "url": "https://github.com/OfficeDev/Office-Add-in-samples/tree/main/Samples/blazor-add-in/outlook-blazor-add-in",
```
Comment (samantharamon) [inline Samples/blazor-add-in/word-blazor-add-in/assets/sample.json:8]:
```suggestion
      "url": "https://github.com/OfficeDev/Office-Add-in-samples/tree/main/Samples/blazor-add-in/word-blazor-add-in",
```

File: Samples/blazor-add-in/word-blazor-add-in/assets/sample.json [hunk 9]
```diff
@@ -0,0 +1,37 @@
...
+      "longDescription": [
+        "This sample shows how to build a Word add-in using .NET Blazor technologies. Blazor Webassembly allows you to build Office Add-ins using .NET, C#, and JavaScript to interact with the Office JS API. The add-in uses JavaScript to work with the document and Office JS APIs, but you build the user interface and all other non-Office interactions in C# and .NET Core Blazor technologies."
```
Comment (samantharamon) [inline Samples/blazor-add-in/word-blazor-add-in/assets/sample.json:10]:
```suggestion
        "This sample shows how to build a Word add-in using .NET Blazor technologies. Blazor WebAssembly allows you to build Office Add-ins using .NET, C#, and JavaScript to interact with the Office JS API. The add-in uses JavaScript to work with the document and Office JS APIs, but you build the user interface and all other non-Office interactions in C# and .NET Core Blazor technologies."
```

**Review by samantharamon — APPROVED** [review]
Suggested link corrections, but other than that, should be good to go!

**Pull request merged**